src/stories/Forms/InputWrapper/index.tsx

```diff
@@ -85,6 +86,10 @@ export const InputWrapper = React.forwardRef(
      * However, if the child is not a valid react component, don't pass props at all.
      */
     const childrenWithGlobalInputProps = React.Children.map(children, (child) => {
+      if (child?.props.children === 'Label') {
```

Comment:
I know this is ugly, but it was the only way I found that checked if it was a label. Up for sugested improvements here!